src/Integration.UnitTests/LocalServices/UnboundProjectFinderTests.cs

```diff
@@ -96,7 +99,7 @@ private class TestConfigurationBuilder
         {
             private readonly Mock<IProjectBinderFactory> projectBinderFactoryMock = new Mock<IProjectBinderFactory>();
             private readonly List<ProjectMock> projects = new List<ProjectMock>();
-            private readonly BindingConfiguration bindingConfiguration = BindingConfiguration.Standalone;
+            private readonly BindingConfiguration bindingConfiguration = new BindingConfiguration(new BoundSonarQubeProject(), SonarLintMode.Connected, null);
```

Comment:
Another problem with the existing test setup: it was using standalone mode, so most of the "find unbound" logic was being skipped.